android/engine/src/main/java/org/smartregister/fhircore/engine/ui/components/register/LoaderView.kt

```diff
@@ -58,7 +58,7 @@ fun LoaderDialog(
   val openDialog = remember { mutableStateOf(true) }
   if (openDialog.value) {
     Dialog(
-      onDismissRequest = { openDialog.value = true },
+      onDismissRequest = { openDialog.value = false },
```

Comment:
What issue is fixed by this?

Reply:
Nice catch. I use this during testing only